src/components/ImageUpload/index.jsx

```diff
@@ -30,14 +32,29 @@ class ImageUpload extends React.Component {
     const {
       className,
       id,
+      input: {
+        name,
+      },
       label,
+      meta: {
+        error,
+        submitFailed,
+      },
     } = this.props;
 
     return (
       <div className={classNames('form-group', className)}>
-        <label htmlFor={id} className="w-100 p-0">{label}</label>  {/* eslint-disable-line jsx-a11y/label-has-for */}
-        <img src={this.state.value} alt="" className="uploaded-image" />
-        <input id={id} type="file" accept="image/*" onChange={this.handleFilePicked} />
+        <div name={name} tabIndex={-1}>
```

Comment:
Didn't want name/tabIndex on the top div?

Reply:
I was trying to keep parity with the way the rest of the DOM looks.

Reply:
![image](https://user-images.githubusercontent.com/12799718/58180385-fe3eed00-7c77-11e9-88ce-c77b5d15cb82.png)